src/components/v5/frame/ColonyHome/partials/DashboardHeader/partials/MembersInformation/MembersInformation.tsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import { Link, useLocation } from 'react-router-dom';
+
+import LoadingSkeleton from '~common/LoadingSkeleton/LoadingSkeleton.tsx';
+import { useMemberContext } from '~context/MemberContext/MemberContext.ts';
+import useGetSelectedDomainFilter from '~hooks/useGetSelectedDomainFilter.tsx';
+import { COLONY_MEMBERS_ROUTE } from '~routes/index.ts';
+import { formatText } from '~utils/intl.ts';
+import UserAvatars from '~v5/shared/UserAvatars/index.ts';
+
+const displayName =
+  'v5.common.ColonyDashboardHeader.partials.MembersInformation';
+
+const MembersInformation = () => {
+  const { search } = useLocation();
+
+  const selectedDomain = useGetSelectedDomainFilter();
+  const nativeDomainId = selectedDomain?.nativeId;
+  const {
+    loading: membersLoading,
+    filteredMembers,
+    followersCount,
+  } = useMemberContext();
+
+  const selectedMembers = nativeDomainId
+    ? filteredMembers.filter(
+        ({ roles, reputation }) =>
+          roles?.items?.find(
+            (role) => role?.domain.nativeId === nativeDomainId,
+          ) ||
```

Comment:
Could we extract this `(role) => role?.domain.nativeId === nativeDomainId` into a more generic function

```
type EntityWithDomainNativeId = {
  domain: Pick<Domain, 'nativeId'>
};

const filterByDomainNativeId = (entity: EntityWithDomainNativeId | null, nativeDomainId) => entity?.domain.nativeId === nativeDomainId
```

And then use it `(role) => filterByDomainNativeId(role, nativeDomainId)`